qa/standalone/ceph-helpers.sh

```diff
@@ -169,11 +172,11 @@ function teardown() {
       pattern=""
     fi
     # Local we start with core and teuthology ends with core
-    if ls $(dirname $pattern) | grep -q '^core\|core$' ; then
+    if ls $(dirname $pattern) | grep -q '^core\|core$' | grep -q "$dir"; then
         cores="yes"
         if [ -n "$LOCALRUN" ]; then
 	    mkdir /tmp/cores.$$ 2> /dev/null || true
-	    for i in $(ls $(dirname $(sysctl -n $KERNCORE)) | grep '^core\|core$'); do
+	    for i in $(ls $(dirname $(sysctl -n $KERNCORE)) | grep '^core\|core$'| grep "$dir"); do
```

Comment:
@tchaikov This does not run with teuthology ($LOCALRUN).  The reason that we don't need to grep for $dir is that the cores are in the upper directory not in the  i.e. td/ of td/TEST_NAME/.